src/world/partners.c

```diff
@@ -39,6 +39,9 @@ extern s32 D_8010CFB8;
 extern s32 D_8010CFBC;
 extern s32 D_8010CFCC;
 
+extern s32 D_801083D8[];
+extern s32 D_80108068[];
```

Comment:
these are out of order - the ones you added should be swapped and come before the CF* ones